### PR TITLE
Update debugging.md

### PR DESCRIPTION
**Pull request by cnizzardini**
Configurations necessary for Mac + docker desktop setups per https://github.com/api-platform/api-platform/issues/796

### CODE REVIEW

File: distribution/debugging.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ services:
       PHP_IDE_CONFIG: serverName=api-platform
 ```
 
+Note for Mac environments use the following:
+
+```yml
+      XDEBUG_CONFIG: >-
+        remote_enable=1
+        remote_host=docker.for.mac.localhost
```
Comment (GregoireHebert) [inline distribution/debugging.md:62]: the real key changer necessary here is this line alone no?

Comment (cnizzardini) [reply distribution/debugging.md:62]: I tested this a few different times locally on my macOs Mojava version 10.14.6 running docker desktop 2.1.0.5. I needed the following configurations for breakpoints to work with PHP Storm:

```yml
        remote_enable=1
        remote_host=docker.for.mac.localhost
        remote_connect_back=0
        remote_port=9000
        idekey=PHPSTORM
```

Comment (cnizzardini) [reply distribution/debugging.md:62]: So it seems the pertinent changes are 

```yml
        remote_host=host.docker.internal
        remote_connect_back=0
```

I am not sure why remote_connect_back must be turned off for it to work on my local, but that is the behavior I am getting.

**Comment by dunglas** [comment]
Thanks @cnizzardini!

**Pull request merged**